osmoutils/accum/accum.go

```diff
@@ -20,20 +20,21 @@ type AccumulatorObject struct {
 	name string
 
 	// Accumulator's current value (pulled from AccumulatorContent)
-	value sdk.Dec
+	value sdk.DecCoins
 }
 
+type PositionOptions struct{}
+
 // Makes a new accumulator at store/accum/{accumName}
 // returns an error if already exists / theres some overlapping keys
 func MakeAccumulator(accumStore store.KVStore, accumName string) error {
-	keybz := []byte(accumName)
-	if accumStore.Has(keybz) {
+	if accumStore.Has(formatAccumPrefixKey(accumName)) {
 		return errors.New("Accumulator with given name already exists in store")
 	}
 
 	// New accumulator values start out at zero
 	// TODO: consider whether this should be a parameter instead of always zero
-	initAccumValue := sdk.ZeroDec()
+	initAccumValue := sdk.NewDecCoins()
 
 	newAccum := AccumulatorObject{accumStore, accumName, initAccumValue}
```

Comment:
Note: I did not find a way to update the store here with an "accum" prefix. Instead, we must use the new "format..." helpers from `prefix.go` every time we want to interact with the store.